tests/make.zunit

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env zunit
+
+@setup {
+    load ../fzf-zsh-completions.plugin.zsh
+    TEMP=$(mktemp -d)
+    pushd $TEMP
+}
+
```

Comment:
This test lacks of teardown section that wipes out the temporary directory.

Reply:
🙆